saleor/app/installation_utils.py

```diff
@@ -12,6 +13,21 @@
 REQUEST_TIMEOUT = 25
 
 
+class AppInstallationError(HTTPError):
+    pass
+
+
+def validate_app_install_response(response: Response):
+    try:
+        response.raise_for_status()
+    except HTTPError as err:
+        try:
+            error_msg = str(response.json()["error"]["message"])
```

Comment:
If the server did not return the message, maybe we can fall back to a generic one.

For example `Server responded with status ${response.status_code}`?

But that's just a suggestion, besides that LGTM.

Reply:
Standard `HTTPError` will be raised if the server will not return an error message. Then [here](https://github.com/saleor/saleor/blob/1e9499fef5cd3077c2dfdcb7796a6525c280e493/saleor/app/tasks.py#L27-L31), this error will be handled, and default message will be set:

> Failed to connect to app. Try later or contact with app support.

When we introduce error context, we could add there response `status_code`.